Formula/leapp-cli.rb

```diff
@@ -0,0 +1,23 @@
+require "language/node"
+
+class LeappCli < Formula
+  desc "Cloud credentials manager cli"
+  homepage "https://github.com/noovolari/leapp"
+  url "https://registry.npmjs.org/@noovolari/leapp-cli/-/leapp-cli-0.1.13.tgz"
+  sha256 "944ce4928c2d2dd562890310ea6f2f96d3ace5f1c3b7e8bbc9ef8323403359d8"
+  license "MPL-2.0"
+
+  depends_on "python@3.9" => :build
+  depends_on "node"
+  depends_on arch: :x86_64
```

Comment:
Why doesn't it work on arm?

Reply:
I'm hitting the `node-gyp` issue. I wrote a little about it in the pr description. I figured it was due to an arm issue so I tried to ignore it here.

Reply:
Same error... @SMillerDev 

```
npm ERR! gyp sill build/config.gypi       'lib/_stream_readable.js',
```

Would you happen to know how to fix this ?

Reply:
Hmm, it seems that `pkg-config` is needed. I added it and waiting for tests to start.

Reply:
Nice, it worked!